cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -913,6 +914,18 @@ function Configure-CniNetworking {
   Log-Output "CNI config:`n$(Get-Content -Raw ${l2bridge_conf})"
 }
 
+# Obtain the host dns conf and save it to a file so that kubelet/CNI can use it to configure
+# dns suffix search list for pods
+function Configure-HostDNSConf {
```

Comment:
Please add to this comment: must be called after Configure-HostNetworkingService

Reply:
Updated.